### DISCUSSION TITLE
Changing maxDuration on Vercel functions doesn't work

### DISCUSSION THREAD

**Discussion by darksipu** [vercel/community#5254] [Help] [answered]
### Page to Investigate

https://vercel.com/docs/functions/configuring-functions/duration#node.js,-next.js-13.5-or-higher,-sveltekit,-and-remix

### Steps to Reproduce

I'm having an issue when trying to extend the maxDuration on the Vercel functions. I'm on Next 14.04, and I've got this code in pages/api/functionName.js

`export const config = { maxDuration: 300 }`

The logs on Vercel still show me a max run of 15s. I'm on the Pro account trial.

I also tried configuring it globally and had no luck.

It's a continuation of this thread: https://github.com/orgs/vercel/discussions/4248#discussioncomment-7310341

**Comment by swarnava** [comment]
Hi,

The above syntax is for `app` directory. Can you confirm if you are applying this in App directory?

**Comment by darksipu** [comment]
Well, this is what the above link on Vercel says for running the app with the pages directory:

<img width="840" alt="Screenshot 2023-12-21 at 16 45 58" src="https://github.com/vercel/community/assets/15602688/c93fab72-b0b1-4dc6-b89a-20f14e9acaae">

The App directory syntax is:
<img width="844" alt="Screenshot 2023-12-21 at 16 55 02" src="https://github.com/vercel/community/assets/15602688/f7d8b31d-e55a-4081-8460-8c9729ea526c">

**Reply by leerob** [reply, marked as answer]
I believe the issue here is the trial account, which has the same limits as the free tier (hobby). Once you are on a paid plan, you can increase the max duration further than the default.
